js/models/Profile.js

```diff
@@ -6,6 +6,10 @@ import SocialAccounts from '../collections/SocialAccounts';
 export default class extends BaseModel {
   defaults() {
     return {
+      name: `ob ${Math.random().toString(36).slice(2)}`,
```

Comment:
If you don't provide a name, we should auto-generate a random one for you, the same as in v1.